jetty-webapp/src/test/java/org/eclipse/jetty/webapp/WebAppContextTest.java

```diff
@@ -76,7 +76,7 @@ public void sessionDestroyed(HttpSessionEvent se)
     @AfterEach
     public void tearDown()
     {
-        Configurations.clearKnown();
+        Configurations.cleanKnown();
```

Comment:
this is a unrelated change.

"clear" (verb) means "to remove unwanted items"
"clean" (verb) means "make (something or someone) free of dirt, marks, or mess, especially by washing, wiping, or brushing" 

Using "clean" here is not correct.